release-notes.md

```diff
@@ -77,22 +85,20 @@ The following updates were made in this release:
 
 The following updates were made in this release:
 
-#### Android
-
-Android Places Extension 1.2.0:
+**Android Places 1.2.0**
 
 * Added a new API to process an individual `Geofence`.
 * Bug fix to prevent multiple consecutive entry events.
 
-Android Places Monitor 1.0.0:
+**Android Places Monitor 1.0.0**
 
-* Initial release of the Places Monitor for Android.
+* Initial release of the Places Monitor for Android. The Places Monitor manages the OS-level Location APIs and communicates directly with the Places Extension.  With both extensions installed, customers can have out-of-the-box region monitoring in their application.
```

Comment:
- Please put the 2nd sentence on a new line.
- At the end of the 2nd sentence, please replace "Extension" with a lower case "e".
- Looks like there might be an extra space between "Extension." and "With". If yes, please remove.